src/ldp/representation/ResourceIdentifier.ts

```diff
@@ -1,4 +1,9 @@
 /**
  * The unique identifier of a resource.
  */
-export interface ResourceIdentifier {}
+export interface ResourceIdentifier {
+  /**
+   * Path to the relevant resource. Usually this would be an URL.
```

Comment:
> Usually this would be an URL.

URI? Can this also be an IRI?

Why don't we name the field `uri`/`iri`?

Reply:
I mention it will usually be an URL because of the restful interface, but true, as a standalone class there is no reason it needs to be limited to that I guess. Reason it's called path is simply because I copied that from [here](https://github.com/RubenVerborgh/solid-server-ts/blob/master/src/ldp/ResourceIdentifier.ts). Probably because that's even more generic.